### PR TITLE
gRPC guide: syntax highlighting for proto example, small fixes

### PR DESCRIPTION
**Pull request by rsvoboda**
gRPC guide:

 - syntax highlighting for proto example using javascript keyword trick, now it is just plain text
 - small fixes: removal of ``` and text to properties metadata change

### CODE REVIEW

File: docs/src/main/asciidoc/grpc-getting-started.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ Alternatively to using the `generate-code` goal of the `quarkus-maven-plugin`, y
 Let's start with a simple _Hello_ service.
 Create the `src/main/proto/helloworld.proto` file with the following content:
 
-[source]
+[source,javascript]
```
Comment (michalszynkiewicz) [inline docs/src/main/asciidoc/grpc-getting-started.adoc:67]: why javascript?

Comment (michalszynkiewicz) [reply docs/src/main/asciidoc/grpc-getting-started.adoc:67]: maybe `[source,protobuf]` will work?

Comment (rsvoboda) [reply docs/src/main/asciidoc/grpc-getting-started.adoc:67]: `protobuf` is not recognized, plain text is rendered. That's why I used `javascript` keyword to get some highlighting.

Comment (michalszynkiewicz) [reply docs/src/main/asciidoc/grpc-getting-started.adoc:67]: oh, I see. 

I checked the available languages and SASS looks interesting:
image

**Review by michalszynkiewicz — APPROVED** [review]

**Pull request merged**